src/LiveDevelopment/MultiBrowserImpl/launchers/node/LauncherDomain.js

```diff
@@ -43,6 +43,18 @@ function _cmdLaunch(url) {
     open(url);
 }
 
+function _launchChromeWithRDP(url, enableRemoteDebugging) {
```

Comment:
copied in from https://github.com/brackets-cont/brackets-shell/blob/master/appshell/appshell_extensions_win.cpp#L352

Reply:
mac: https://github.com/brackets-cont/brackets-shell/blob/master/appshell/appshell_extensions_mac.mm#L285